### PR TITLE
Updated Polish translation

### PR DESCRIPTION
**Pull request by Witia1**

### CODE REVIEW

File: locale/pl/LC_MESSAGES/javascript.po [hunk 1]
```diff
@@ -32,7 +33,8 @@ msgstr "Wybierz plik…"
 
 #: static/js/common/upload-addon.js:60
 msgid "Your add-on should end with .xpi, .jar or .xml"
-msgstr "Nazwa pliku dodatku powinna kończyć się rozszerzeniem .xpi, .jar lub .xml"
+msgstr ""
+"Nazwa pliku dodatku powinna kończyć się rozszerzeniem .xpi, .jar lub .xml"
```
Comment (magopian) [inline locale/pl/LC_MESSAGES/javascript.po]: Is it usual to have the string split like that? @clouserw ?

Comment (clouserw) [inline locale/pl/LC_MESSAGES/javascript.po]: Shouldn't matter.  That's just how it rolls at a certain length.

Comment (magopian) [inline locale/pl/LC_MESSAGES/javascript.po]: Should we merge then? I know you were usually doing it in the past, should I now do it? Or @andymckay ?
Are there things to look for, double check, be aware of, be cautious of?

Comment (andymckay) [inline locale/pl/LC_MESSAGES/javascript.po]: I just did a sanity check that the only change is the .po file and not sneaked in an OMG security hole.

**Pull request merged**